doc/sphinx-guides/source/developers/aux-file-support.rstnew

```diff
@@ -0,0 +1,39 @@
+Auxiliary File Support
+======================
+
+Auxiliary file support is experimental. Auxiliary files in the Dataverse Software are being added to support depositing and downloading differentially private metadata, as part of the OpenDP project (opendp.org). In future versions, this approach may become more broadly used and supported. 
+
+Adding an Auxiliary File to a Datafile
+--------------------------------------
+To add an auxiliary file, specify the primary key of the datafile (FILE_ID), and the formatTag and formatVersion (if applicable) associated with the auxiliary file. There are multiple form parameters. "Origin" specifies the application/entity that created the auxiliary file, and "isPublic" controls access to downloading the file. If "isPublic" is true, any user can download the file if the dataset has been published, else, access authorization is based on the access rules as defined for the DataFile itself. The "type" parameter can be "DP" (for Differentially Private Statistics), "OTHER" (this will be selected for you if don't specify a type), or a type of your own choosing (which will be displayed along side "OTHER" auxiliary files until a friendly name is added to a properties file).
```

Comment:
can we not use "Other" as a type and just have it be blank if they don't want to specify?

Reply:
My bad, this was an old version of the aux file page that I accidentally committed. Removed in 51cffe8b1.